pluginconfig/execute.go

```diff
@@ -1,35 +1,89 @@
 package pluginconfig
 
 import (
+	"encoding/json"
+	"errors"
 	"time"
 
-	"github.com/smartcontractkit/libocr/commontypes"
-
+	commonconfig "github.com/smartcontractkit/chainlink-common/pkg/config"
 	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
 )
 
 type ExecutePluginConfig struct {
 	// DestChain is the ccip destination chain configured for the execute DON.
 	DestChain cciptypes.ChainSelector `json:"destChain"`
 
-	// ObserverInfo is a map of oracle IDs to ObserverInfo.
-	ObserverInfo map[commontypes.OracleID]ObserverInfo `json:"observerInfo"`
-
 	// SyncTimeout is the timeout for syncing the commit plugin reader.
 	SyncTimeout time.Duration `json:"syncTimeout"`
 
 	// SyncFrequency is the frequency at which the commit plugin reader should sync.
 	SyncFrequency time.Duration `json:"syncFrequency"`
 
+	// OffchainConfig is the offchain config set for the exec DON.
+	OffchainConfig ExecuteOffchainConfig `json:"offchainConfig"`
```

Comment:
nice, I was wondering if they overlapped completely like this.

You could fully embed it as well, most off the shelf decoders will append changes rather than overwrite them. Not sure if our custom decoders work that way.
```suggestion
	// ExecuteOffchainConfig is the offchain config set for the exec DON.
	ExecuteOffchainConfig`
```